src/packages/client/src/__tests__/runtime-tests/middlewares/index.js

```diff
@@ -7,13 +7,24 @@ module.exports = async () => {
   const allResults = []
   const engineResults = []
 
```

Comment:
It's missing a `db.use('all'` I think (The default is all though.)

Reply:
We don't want to document `db.use('all'` yet, hence no test yet.